app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailFragment.kt

```diff
@@ -77,13 +78,22 @@ class SessionDetailFragment : Fragment(), Injectable {
         binding.toolbar.setNavigationOnClickListener { activity?.finish() }
     }
 
-    private fun bindSession(session: Session.SpeechSession) {
+    private fun bindSession(session: Session.SpeechSession, sessionId: String) {
```

Comment:
I think you can get sessionId by `session.id`.
What do you think? 👀

Reply:
Oops, I'll fix